File: ch2_labs/lab5_C#/TextLibrary/Processor.cs
using System;

namespace TextLibrary
{
    public static class Processor
    {
        public static void ProcessRyadok(Ryadok r)
        {
            Console.WriteLine($"До змін: {r.Text}, Довжина: {r.GetLength()}");
            r.Modify();
            Console.WriteLine($"Після змін: {r.Text}, Довжина: {r.GetLength()}");
        }
    }
}

File: ch2_labs/lab5_C++/src/Processor.cpp
#include "Processor.h"

void processRyadok(Ryadok* r) {
    std::cout << "До змін: " << r->getText() << ", Довжина: " << r->getLength() << std::endl;
    r->modify();
    std::cout << "Після змін: " << r->getText() << ", Довжина: " << r->getLength() << std::endl;
}
